turn_by_turn/io.py

```diff
@@ -2,7 +2,7 @@
 IO
 --
 
-This module contains high-level I/O functions to read and write tur-by-turn data objects in different
+This module contains high-level I/O functions to read and write turn-by-turn data objects in different
```

Comment:
Writing it on the only modified line: let's also add the new reader module in here (lower, in `TBT_MODULES` and `WRITERS`)